integration/README.md

```diff
@@ -43,6 +53,9 @@ Optional fields are:
 * `inrupt.test.webid` is needed only if we want to run the integration tests on a live service. Otherwise, this property needs to be left out because it will be populated by the Mocked services with a mock username called `someuser`.
 * `inrupt.test.requester.webid` is only needed in the access grants test scenarios and can be also left empty because the Mocked services will create a username called `requester`.
 * `inrupt.test.auth-method` refers to the [client authentication](https://openid.net/specs/openid-connect-core-1_0.html#ClientAuthentication) method and has a default value of `client_secret_basic`. This value is used when this property is not provided.
+* `inrupt.test.feature.request-metadata` is a `true` or `false` config that signals if the feature is available on the server and only then runs the according scenarios. By default, this is set on `false`.
+* `inrupt.test.request-metadata-headers-that-propagate` if the above is `true`, the test scenarios need to know which headers the servers can propagate. This is a string comma separated list.  
```

Comment:
Wouldn't it be more consistent to have `inrupt.test.feature.request-metadata` (as it currently is) and `inrupt.test.feature.request-metadata.headers-that-propagate`?

Reply:
I was trying to make a difference between - flag for feature active or not and then the headers.